web-ng/src/app/services/project/project.service.ts

```diff
@@ -120,4 +125,16 @@ export class ProjectService {
     const acl = this.getCurrentProjectAcl();
     return !!acl.find(entry => entry.email === userEmail && entry.isManager());
   }
+
+  /**
+   * Returns the acl of the project.
+   */
+   getProjectAcl(project: Project): AclEntry[] {
```

Comment:
Can you explain why this method is added ? It is not used as far as I can see.

Reply:
It will be used later.

Reply:
"the project" -> "the specified project"